pkg/cli/cmd/app/status/objectformats.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import "github.com/radius-project/radius/pkg/cli/output"
+
+// statusFormat sets up the columns and headings for a table to display application names and resource counts.
+func statusFormat() output.FormatterOptions {
```

Comment:
I moved formats to the most specific package that I could. A lot of these formats were defined to output tables for private data types, which meant they could not be tested without **some** kind of refactor. 

It made way more sense to keep the functionality in the packages that need it.